sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -576,18 +578,19 @@ def get_tpu_df():
         action='store_true',
         help='Fetch all global regions, not just the U.S. ones.')
     args = parser.parse_args()
-    region_prefix = ALL_REGION_PREFIX if args.all_regions else US_REGION_PREFIX
+    region_prefix_filter = ALL_REGION_PREFIX if args.all_regions else US_REGION_PREFIX
 
-    vm_df = get_vm_df(region_prefix)
-    gpu_df = get_gpu_df(region_prefix)
-    tpu_df = get_tpu_df()
-    catalog_df = pd.concat([vm_df, gpu_df, tpu_df])
+    processed_vm_df = get_vm_df(region_prefix_filter)
+    processed_gpu_df = get_gpu_df(region_prefix_filter)
+    processed_tpu_df = get_tpu_df()
```

Comment:
@Michaelvll BTW, What was the reason for this renaming?

Reply:
Mainly due to the formatter telling me that I can't re-define the variable globally when it appears inside those functions.

Reply:
Got it. Thanks.